aspnet/signalr/overview/guide-to-the-api/hubs-api-guide-server.md

```diff
@@ -569,6 +572,7 @@ In VB.NET or in a strongly-typed hub, the caller state object can't be accessed
 To handle errors that occur in your Hub class methods, use one or more of the following methods:
 
 - Wrap your method code in try-catch blocks and log the exception object. For debugging purposes you can send the exception to the client, but for security reasons sending detailed information to clients in production is not recommended.
+- Use `await` when invoking client methods to ensure that any exceptions that occur while trying to send the message are observed and can be caught in a try-catch block.
```

Comment:
The lead-in to the bullets says "use one or more" of these methods, which implies that any one of them is optional. If only the try-catch method depends on use of await, you could incorporate the await text into that bullet instead of making it a separate one.  If await is always required for all of these methods, it might be better to take it out of the bullet list and say that all methods depend on it.